llm_memory/utils/user_profile.py

```
@@ -27,17 +27,21 @@ def normalize_tags(tags: Iterable[str]) -> List[str]:
-    规则：长度 > 5，且不是纯中文、不是纯英文字母。
-    纯数字、数字+字母混合、含下划线/连字符的平台标识符等都算用户 ID。
+    排除法规则：长度 > 5，且不含中文、不含空白、不是纯英文词形。
+    纯数字、数字+字母混合、含下划线/连字符且含数字的平台标识符等算用户 ID。
+    描述性标签（含中文、含空格、纯英文词形）一律不算。
     """
     text = str(tag or "").strip()
     if len(text) <= 5:
         return False
-    # 纯中文 → 不是 ID
-    if all("\u4e00" <= c <= "\u9fff" for c in text):
+    # 含中文字符 → 描述性标签，不是平台 ID（如「1998年10月19日」）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)